### PR TITLE
fix: make sure aria-selected gets correctly populated in production build LAB-7191

### PR DESCRIPTION
**Pull request by nafissa-hassan**

### CODE REVIEW

File: addon/components/power-select.hbs [hunk 1]
```diff
@@ -176,7 +177,6 @@
             id={{listboxId}}
             @ariaLabel={{@ariaLabel}}
             @ariaLabelledBy={{@ariaLabelledBy}}
-            @ariaSelected={{ember-power-select-is-selected opt @selected}}
```
Comment (nafissa-hassan) [inline addon/components/power-select.hbs:179]: we didn't have access to the opt value here, so I had to do the calculation one level down in the options component

File: addon/components/power-select/options.hbs [hunk 2]
```diff
@@ -17,20 +17,20 @@
           <Options
             @options={{opt.options}}
             @select={{@select}}
+            @selected={{@selected}}
```
Comment (nafissa-hassan) [inline addon/components/power-select/options.hbs:20]: passing through the currently selected value

File: addon/components/power-select/options.hbs [hunk 3]
```diff
@@ -17,20 +17,20 @@
...
-            @ariaSelected={{@ariaSelected}}
             class="ember-power-select-options" as |option|>
             {{yield option @select}}
           </Options>
         </Group>
       {{else}}
         <li class="ember-power-select-option"
           id="{{@select.uniqueId}}-{{@groupIndex}}{{index}}"
-          aria-selected={{ember-power-select-is-selected opt @select.selected}}
+          aria-selected="{{ember-power-select-is-selected opt @selected}}"
```
Comment (nafissa-hassan) [inline addon/components/power-select/options.hbs:33]: doing the calculation here

File: addon/components/power-select.hbs [hunk 4]
```diff
@@ -167,6 +167,7 @@
           <Options
             @loadingMessage={{or @loadingMessage "Loading options..."}}
             @select={{publicAPI}}
+            @selected={{@selected}}
```
Comment (nafissa-hassan) [inline addon/components/power-select.hbs:170]: passing through the currently selected value

**Review by RTrofymenko — APPROVED** [review]

**Pull request merged**